backend/utils.py

```diff
@@ -17,6 +18,8 @@
     ".webm": lambda file: Whisperexporter().whisper_to_text(file.file, filename=file.filename, file_meta=file),
     ".m4a": lambda file: Whisperexporter().whisper_to_text(file.file, filename=file.filename, file_meta=file),
     ".docx": lambda file: load_docx(file.file, filename=file.filename, file_meta=file),
+    ".xlsx": lambda file: load_xlsx(file.file, filename=file.filename, file_meta=file),
+    ".csv": lambda file: load_txt(file.file, filename=file.filename, file_meta=file),
```

Comment:
Please test if `.xls` also works

Reply:
No, it throws error with `.xls` file. I think we'll need to use [xlrd](https://xlrd.readthedocs.io/en/latest/) to work with `.xls` file.

Reply:
Would xlrd work with all kinds? Then I suggest using it

Reply:
Sadly `xlrd` only works with `xls`
> This library will no longer read anything other than .xls files. For alternatives that read newer file formats, please see http://www.python-excel.org/

And [openpyxl](https://openpyxl.readthedocs.io/en/stable/) only works with `xlsx` files.
Could I address `xls` file in another issue/PR then?